usr/share/rear/init/default/100_check_stale_nfs_mounts.sh

```diff
@@ -0,0 +1,11 @@
+# 100_check_stale_nfs_mounts.sh
+# Purpose is to have a simple test if we have stale NFS mount points which
+# could lead to a hangind ReaR session.
+# In case there is a stale NFS present we bail out with an error to get it fixed before
+# running ReaR again.
+# See issue https://github.com/rear/rear/issues/3109
+cat /proc/mounts | grep -i "nfs" | \
+    while read exported_fs mount_point junk ;
```

Comment:
I think `grep -i "nfs"` may match too many false positives.

E.g. on my homeoffice workstation I get:
```
# cat /proc/mounts | grep -i "nfs"
nfsd /proc/fs/nfsd nfsd rw,relatime 0 0
sunrpc /var/lib/nfs/rpc_pipefs rpc_pipefs rw,relatime 0 0

# cat /proc/mounts | grep -i "nfs" | while read exported_fs mount_point junk ; do echo "exported_fs $exported_fs at mount_point $mount_point" ; done
exported_fs nfsd at mount_point /proc/fs/nfsd
exported_fs sunrpc at mount_point /var/lib/nfs/rpc_pipefs

# cat /proc/mounts | grep -i "nfs" | while read exported_fs mount_point junk ; do timeout 5 df $mount_point >/dev/null  && echo $mount_point OK || echo $mount_point FAIL ; done
/proc/fs/nfsd OK
/var/lib/nfs/rpc_pipefs OK
```
Perhaps it does not matter in practive
when more are tested with 'df' than needed?

But for some the 'df' test fails for me
```
# cat /proc/mounts | while read exported_fs mount_point junk ; do timeout 5 df $mount_point >/dev/null  && echo $mount_point OK || echo $mount_point FAIL ; done | grep -v OK
df: /run/user/1000/gvfs: Permission denied
/run/user/1000/gvfs FAIL
df: /run/user/1000/doc: Permission denied
/run/user/1000/doc FAIL
df: /root/.cache/doc: Operation not permitted
/root/.cache/doc FAIL
```
and when by accident such one contains 'nfs' (ignore case)
we would falsely error out.

Reply:
@jsmeix  Thank you for the useful comments/tests. I agree and therefore, I improved the grep to avoid possible false positives.